src/main/java/uk/gov/companieshouse/api/accounts/validation/transactionclosure/StocksTxnClosureValidator.java

```diff
@@ -0,0 +1,81 @@
+package uk.gov.companieshouse.api.accounts.validation.transactionclosure;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.api.accounts.enumeration.AccountingNoteType;
+import uk.gov.companieshouse.api.accounts.exception.DataException;
+import uk.gov.companieshouse.api.accounts.links.SmallFullLinkType;
+import uk.gov.companieshouse.api.accounts.model.rest.BalanceSheet;
+import uk.gov.companieshouse.api.accounts.model.rest.CurrentAssets;
+import uk.gov.companieshouse.api.accounts.model.rest.Note;
+import uk.gov.companieshouse.api.accounts.model.rest.SmallFull;
+import uk.gov.companieshouse.api.accounts.model.validation.Errors;
+import uk.gov.companieshouse.api.accounts.service.CompanyService;
+import uk.gov.companieshouse.api.accounts.service.NoteService;
+import uk.gov.companieshouse.api.accounts.service.response.ResponseObject;
+import uk.gov.companieshouse.api.accounts.validation.BaseValidator;
+import uk.gov.companieshouse.api.accounts.validation.NoteValidator;
+import uk.gov.companieshouse.api.accounts.validation.NoteValidatorFactory;
+import uk.gov.companieshouse.api.model.transaction.Transaction;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Optional;
+
+@Component
+public class StocksTxnClosureValidator extends BaseValidator {
+
+    private final NoteService stocksNoteService;
+
+    private final NoteValidatorFactory<Note> validatorFactory;
+
+    private static final String SMALL_FULL_CURRENT_STOCKS = "$.company_accounts.small_full.current_period.notes.stocks";
```

Comment:
I think this path is the wrong format; I believe it ought to be 
```
$.company_accounts.small_full.notes.stocks.current_period
```
to follow the resource structure